src/k8s-engine/notimplemented.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
+import { ContainerEngine } from '../config/settings';
+
 const events = require('events');
 const { dialog } = require('electron');
 const { State } = require('./k8s');
 
 /**
  * OSNotImplemented is a class for the case that a platform is not implemented.
  */
-class OSNotImplemented extends events.EventEmitter {
+export class OSNotImplemented extends events.EventEmitter {
```

Comment:
Maybe we should convert this to TypeScript at some point… not in this PR, though.

Reply:
I got compiler errors from the other Backend implementations with the old separate export line.